app/views/sufia/admin/admin_sets/_form.html.erb

```diff
@@ -24,6 +24,10 @@
                   <%= f.input :thumbnail_id, collection: @form.select_files %>
                 <% end %>
 
+                <% if @form.persisted? %>
```

Comment:
But we want the permission template to be created as soon as the AdminSet is created (and we want to persist the workflow_name on it).